06.bowling_object/frame.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require_relative 'shot'
+
+class Frame
+  attr_reader :shots
+
+  LAST_FRAME = 10
+
+  def initialize(marks)
+    @shots = marks.map { |mark| Shot.new(mark) }
+  end
+
+  def score
+    @shots.sum(&:score)
+  end
+
+  def strike?
+    @shots.first.strike?
+  end
+
+  def spare?
+    @shots[0].score + @shots[1].score == 10
```

Comment:
```suggestion
    !strike? && @shots[0].score + @shots[1].score == 10
```

念のためstrikeとspareは区別した方が良さそう

その上で、

```suggestion
    !strike? && @shots[0..1].sum(&:score) == 10
```

というふうにsumを使うと少しロジックがシンプルになります